docs/docs/workflows/sbom.md

```diff
@@ -1,5 +1,9 @@
 # Consume software bill of materials (SBOMs)
 
+import AsciinemaWidget from '../../src/components/AsciinemaWidget';
+
+<AsciinemaWidget src="/constellation/assets/check-sbom.cast" fontSize={16} rows={18} cols={80} idleTimeLimit={3} preload={true} />
```

Comment:
```suggestion
<asciinemaWidget src="/constellation/assets/check-sbom.cast" fontSize={16} rows={18} cols={80} idleTimeLimit={3} preload={true} />
```
If you want to use this on multiple pages, you can adjust it like this and add the following lines at the respective places in `src/theme/MDXComponents.js`:
```js
import AsciinemaWidget from '@site/src/components/AsciinemaWidget';
```
```js
    asciinemaWidget: AsciinemaWidget,
```

Reply:
Awesome that makes it a lot cleaner, thanks 🙂